android/app/src/main/java/com/microsoft/codepush/common/apiRequests/DownloadPackageTask.java

```diff
@@ -20,48 +20,36 @@
 /**
  * Downloads an update.
  */
-public class DownloadPackageJob extends BaseHttpJob<CodePushDownloadPackageResult> {
+public class DownloadPackageTask extends BaseHttpTask<CodePushDownloadPackageResult> {
 
     /**
      * Header in the beginning of every zip file.
      */
-    private final Integer ZIP_HEADER = 0x504b0304;
-
-    /**
-     * Url for downloading an update.
-     */
-    private String downloadUrlString;
+    private final static Integer ZIP_HEADER = 0x504b0304;
 
     /**
      * Path to download file to.
      */
-    private File downloadFile;
+    private File mDownloadFile;
 
     /**
      * Callback for download process.
      */
-    private DownloadProgressCallback downloadProgressCallback;
-
-    /**
-     * Creates instance of {@link DownloadPackageJob}.
-     *
-     * @param fileUtils instance of {@link FileUtils} to work with.
-     */
-    public DownloadPackageJob(FileUtils fileUtils) {
```

Comment:
So if you removed this constructor, where are `mFileUtils` assigned?

Reply:
@annakocheshkova here: https://github.com/Microsoft/react-native-code-push/pull/1190/files/f9d71f7d2aa31058d35c5b85265d946cdf869caa#diff-17e6f8e9bf1806dd28225e4928e5c8b0R48